### PR TITLE
[Method References 2] LPS-164453 Convert method reference to ServiceTrackerMap in module dynamic-data-mapping-data-provider-web

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I have removed the unused method.

Detail: https://github.com/liferay-core-infra/liferay-portal/pull/1891#event-7538781359

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results ? Thanks!

**Comment by W2jx** [comment]
Force-Push is just to rebase the code.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/DDMDataProviderDisplayTracker.java [hunk 1]
```diff
@@ -38,10 +36,6 @@ public DDMDataProviderDisplay getDDMDataProviderDisplay(String portletId) {
 		return _serviceTrackerMap.getService(portletId);
 	}
 
-	public List<DDMDataProviderDisplay> getDDMDataProviderDisplays() {
-		return _getDDMDataProviderDisplays();
-	}
-
 	public String[] getPortletIds() {
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-web/src/main/java/com/liferay/dynamic/data/mapping/data/provider/web/internal/display/DDMDataProviderDisplayTracker.java:39]: @W2jx It seems `getPortletIds()` is also not used by anyone. Please remove it.

**Comment by dantewang** [comment]
@W2jx Since there are too many PRs open, I'm going to close this one. Please resend.

**Pull request closed without merging**